check-before-deployment.md

```diff
@@ -293,6 +293,8 @@ To check whether the NTP service is installed and whether it synchronizes with t
         506 Cannot talk to daemon
         ```
 
+    - If the offset seems to high you may want to run `chronyc makestep` to immediately correct any time offset. Without running this `chronyd` will gradually correct the time offset.
```

Comment:
```suggestion
    - If the offset appears to be too high, you can run the `chronyc makestep` command to immediately correct the time offset. Otherwise, `chronyd` will gradually correct the time offset.
```